cmd/clusternet-hub/app/app.go

```diff
@@ -71,8 +71,6 @@ func NewClusternetHubCmd(ctx context.Context) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.BoolVar(&opts.TunnelLogging, "enable-tunnel-logging", opts.TunnelLogging, "Enable tunnel logging")
```

Comment:
Why removing this?

Reply:
since we don't introduce new options for hub, I will move it back.